core/src/migrations/000081-profilesToRecords.ts

```diff
@@ -97,6 +97,10 @@ export default {
 
     await queryInterface.renameTable("profileProperties", "recordProperties");
 
+    await queryInterface.sequelize.query(
+      `UPDATE "logs" SET "topic"='grouparooRecord' WHERE "topic"='profile'`
```

Comment:
I don't think this was the full list.
`UPDATE "logs" SET "topic"='recordProperty' WHERE "topic"='profileProperty'` at least.